src/power.rs

```diff
@@ -548,6 +553,95 @@ impl TdpLimitManager for LenovoWmiTdpLimitManager {
     }
 }
 
+impl ZotacZoneTdpLimitManager {
+    const PREFIX: &str = "/sys/class/firmware-attributes/zotac-zone-platform/attributes";
```

Comment:
This should be `/sys/class/firmware-attributes/zotac_zone_platform/attributes` ...